src/charts.ts

```diff
@@ -101,6 +104,36 @@ function openMbtilesFile(file: string, filename: string) {
   )
 }
 
+export function encStyleToId(filename: string) {
+  return filename.replace('.json', '').replaceAll(' ', '-').toLocaleLowerCase()
+}
+
+async function openMbstylesFile(file: string, filename: string) {
```

Comment:
I'd like to rename this to `openMapboxStylesFile`- while the `mb` in `mbtiles` is originally also `MapBox` that format has evolved to be a common name, there is no equivalent `mbstyles`.

Reply:
Renamed to `openMapboxStylesFile`